box64-bundle-x86-libs.sh

```diff
@@ -1,121 +1,123 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Is this script actually portable any working on other shells? I used to have a large project written in Zsh and there were lots of issues with compatibility between Bash that I could not solve. In this case, we can guarantee compatibility with Bash, at least.

Reply:
yes, actually I'm a zsh user. Beside, I tested with dash which I have on my system as well. So tested successfully with bash, dash and zsh.